relayer/src/worker.rs

```diff
@@ -91,9 +91,16 @@ impl<ChainA: ChainHandle + 'static, ChainB: ChainHandle + 'static> Worker<ChainA
         debug!("spawning worker for object {}", object.short_name(),);
 
         let worker = match &object {
-            Object::Client(client) => {
-                Self::Client(id, ClientWorker::new(client.clone(), chains, cmd_rx))
-            }
+            Object::Client(client) => Self::Client(
+                id,
+                ClientWorker::new(
+                    client.clone(),
+                    chains,
+                    cmd_rx,
+                    config.mode.clients.misbehaviour,
+                    config.mode.clients.refresh,
```

Comment:
How about we pass the `Packets` struct directly in order to avoid potential issues with [boolean blindness](https://github.com/ostera/reason-design-patterns/blob/master/patterns/boolean-blindness.md)?

Reply:
https://github.com/informalsystems/ibc-rs/pull/1539/commits/a12be44bd4852b74c0d5fb936d6ff9021e96f96a